packages/flutter/lib/src/widgets/visibility.dart

```diff
@@ -173,6 +173,9 @@ class Visibility extends StatelessWidget {
   ///
   /// If this property is true, [Opacity] is used instead of [Offstage].
   ///
+  /// If this property is true, then both [maintainAnimation] and
```

Comment:
This seems to just be a duplication of the sentence in line 164? Remove one of them?

Reply:
Thanks for the advice. I will do that.